### PR TITLE
Add algorithm for computing checksums.

### PR DESCRIPTION
**Pull request by garretrieger**

### CODE REVIEW

File: Overview.bs [hunk 1]
```diff
@@ -528,19 +531,75 @@ Procedures {#procedures}
...
+```
+
+Note: a C implementation of fast hash can be found
+<a href="https://github.com/ztanml/fast-hash">here</a>.
 
 ### Codepoint Reordering ### {#codepoint-reordering}
 
-TODO(garretrieger): write this.
+A codepoint reordering for a font defines a function which maps unicode codepoint values from the
+font to a continuous space of [0, number of codepoints in the font). This transformation is intended
```
Comment (vlevantovsky) [inline Overview.bs:570]: I guess this is a typo, should be ']'

**Review by vlevantovsky — APPROVED** [review]
LGTM

File: Overview.bs [hunk 2]
```diff
@@ -528,19 +531,75 @@ Procedures {#procedures}
...
 
-TODO(garretrieger): write this.
+#### Codepoint Reordering Checksum #### {#reordering-checksum}
 
-#### Recommended algorithm #### {#reordering-algorithm}
+A checksum of a codepoint reordering can be computed as follows:
 
-TODO(garretrieger): write this.
+```
+SEED = 0x11743e80f437ffe6
```
Comment (svgeesus) [inline Overview.bs:585]: It might be useful to explain where the magic numbers come from

**Review by svgeesus — APPROVED** [review]
Looks good

**Pull request merged**